draft-ietf-rats-eat.md

```diff
@@ -222,28 +228,27 @@ Here are some examples:
 
 No single set of claims can accommodate all use cases so EAT is constructed as a framework for defining specific attestation tokens for specific use cases.
-In particular, EAT provides a profile mechanism to be able to clearly specify the claims needed, the cryptographic algorithms that should be used and other for a particular token and use case.
+In particular, EAT provides a profile mechanism to be able to clearly specify the claims needed, the cryptographic algorithms that should be used, and other characteristics for a particular token and use case.
+{{profiles}} describes profile contents and provides a profile that is suitable for constrained device use cases.
 
-The entity side of an EAT implementation generates the claims and typically signs them with an attestation key.
+The entity's EAT implementation generates the claims and typically signs them with an attestation key.
 It is responsible for protecting the attestation key.
 Some EAT implementations will use components with very high resistance to attack like TPMs or secure elements.
-Other may rely solely on simple SW defenses.
+Others may rely solely on simple software defenses.
 
 Nesting of tokens and claims sets is accommodated for composite devices that have multiple subsystems.
 
 An EAT may be encoded in either JSON {{RFC8259}} or CBOR {{RFC8949}} as needed for each use case.
 EAT is built on CBOR Web Token (CWT) {{RFC8392}} and JSON Web Token (JWT) {{RFC7519}} and inherits all their characteristics and their security mechanisms.
-
+Like CWT and JWT, EAT does not imply any message flow.
 
 ## Entity Overview
 
 The document uses the term "entity" to refer to the target of an EAT. Many of the claims defined in this document are claims about an entity, which is equivalent to an attesting environment as defined in [RATS.architecture]. An entity may be the whole device, a subsystem, a subsystem of a subsystem, etc.
 Correspondingly, the EAT format allows claims to be organized using mechanisms like submodules and nested EATs (see {{submods}}).
 The entity to which a claim applies is the submodule in which it appears, or to the top-level entity if it doesn't appear in a submodule.
 
-An entity also corresponds to a "system component", as defined in the Internet Security Glossary {{RFC4949}}.
-That glossary also defines "entity" and "system entity" as something that may be a person or organization as well as a system component.
-In the EAT context, "entity" never refers to a person or organization. The hardware and software that implement a server or service used by a web site may be an entity, but the organization that runs the web site is not an entity nor is the web site itself. An entity is an implementation in hardware, software or both.
+An entity also corresponds to a "system component", as defined in the Internet Security Glossary {{RFC4949}}, except that in the EAT context an "entity" never refers to a person or organization.
```

Comment:
https://github.com/ietf-rats-wg/eat/pull/387

Reply:
I am assuming you are saying "leave it as-is and #387 will apply changes I prefer".